### PR TITLE
Fix whole of style (Extends #361)

### PR DESCRIPTION
**Pull request by Ikuyadeu**
**What problem did you solve?**

#361 was too large to solve all issues.
This PR only apply the easy fix, and I gave up that fixing all rule violation once, especially for the `any` value warnings.
I want to fix remained violations in another issue, after merge this.

I confirmed that fixing current new functions source code readability is very hard work.
This PR can be the first step to keeping this extension maintainability

Next TODO:
* Refactor any values it can be object
  * extension.ts
  * rHelpPanel.ts
  * RHelpProviderBuiltin.ts
  * rstudioapi.ts
  * session.ts
  * test/suite/index.ts

* Simplify extension.ts
  * Remove goto label it can be simple for statement
  * Move Rmd functions to another file

### CODE REVIEW

**Comment by Ikuyadeu** [comment]
Could you review this?
I think this change has no impact on the code behavior, but I want to someones check.

File: src/test/suite/extension.test.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ suite('Extension Tests', () => {
             y
         }
         `.split('\n');
-        function f(i) {return (doc[i]); }
+        function f(i: number) { return (doc[i]); }
```
Comment (renkun-ken) [inline src/test/suite/extension.test.ts:23]: Here on my side, eslint complains about the use of the deprecated `assert.equal`:

image

Should we do anything about it?

**Comment by renkun-ken** [comment]
I go through the changes and all look good. Tested on macOS and everything works well.

File: src/rHelpPanel.ts [hunk 2]
```diff
@@ -1,5 +1,10 @@
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
 
-import * as vscode from 'vscode';
+import { commands, window, QuickPickItem, Uri, Webview, WebviewPanel, WebviewOptions, WebviewPanelOnDidChangeViewStateEvent, ViewColumn } from 'vscode';
```
Comment (ManuelHentschel) [inline src/rHelpPanel.ts:7]: Is there a performance benefit from importing each item explicitly or is this just a stylistic choice?

File: src/test/suite/extension.test.ts (hunk 1 above)
Comment (Ikuyadeu) [reply src/test/suite/extension.test.ts:23]: Thank you for your point.
Maybe it is not eslint, it is a typescript function alert.
But It's good timing, I'll fix it to `assert.strictEqual`

File: src/rHelpPanel.ts (hunk 2 above)
Comment (Ikuyadeu) [reply src/rHelpPanel.ts:7]: It is not alerted by ESLint, but it is useful to refactoring based on the vscode API.
In this case, we can confirm  `rHelpPanel.ts` manage the Webview API related function.
If someone implements a related function in another file, we should combine that in this file or implement super-class

**Comment by Ikuyadeu** [comment]
I merge this, thank you @renkun-ken and @ManuelHentschel for your comments.

**Pull request merged**